doc/guides/cli/poseidonos-cli_volume_set-property.md

```diff
@@ -0,0 +1,52 @@
+## poseidonos-cli volume set-property
+
+Set the properties of a volume.
+
+### Synopsis
+
+
+Set the properties of a volume.
+
+Syntax: 
+	poseidonos-cli volume set-property (--volume-name | -v) VolumeName 
+	(--array-name | -a) ArrayName (--new-volume-name | -n) VolumeName
+	[--primary-volume] [--secondary-volume]
+
+Example: 
+	poseidonos-cli volume set-property --volume-name Volume0 --array-name volume0 
+	--new-volume-name NewVolume0 --user-volume --primary-volume
```

Comment:
user-volume parameter is not explained in the Options below

Reply:
Thanks for the comment. I've removed the --user-volume flag, which is not used anymore.